File: src/main/java/com/github/thorben/webnd/domain/character/AttributeTypeRepository.java
package com.github.thorben.webnd.domain.character;

import com.github.thorben.webnd.domain.games.GameType;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface AttributeTypeRepository extends CrudRepository<AttributeType, Integer> {

	List<AttributeType> findAllByGameType(GameType gameType);

}

File: src/main/kotlin/com/github/thorben/webnd/domain/character/AttributeTypeRepository.kt
package com.github.thorben.webnd.domain.character

import com.github.thorben.webnd.domain.games.GameType
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface AttributeTypeRepository : CrudRepository<AttributeType, Int> {
	fun findAllByGameType(gameType: GameType): List<AttributeType>
}